File: projeto/back/mais-consulta/src/main/java/com/bandtec/mais/consulta/gateway/database/repository/UsuarioRepository.java
package com.bandtec.mais.consulta.gateway.database.repository;

import com.bandtec.mais.consulta.domain.Usuario;
import org.springframework.data.jpa.repository.JpaRepository;

public interface UsuarioRepository extends JpaRepository<Usuario, Long> {

    Usuario findByLoginAndPassword (String login, String senha);

}

File: projeto/back/maisconsulta/src/main/kotlin/com/bandtec/maisconsulta/gateway/database/repository/UsuarioRepository.kt
package com.bandtec.maisconsulta.gateway.database.repository

import com.bandtec.maisconsulta.domain.Usuario
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository("usuarioRepository")
interface UsuarioRepository : JpaRepository<Usuario, Long> {

    fun findByLoginAndPassword(login: String, senha: String): Usuario

}